WasatchNET/EEPROM.cs

```diff
@@ -1737,11 +1743,11 @@ public void setDefault(Spectrometer a)
 
             startupIntegrationTimeMS = 0;
             double temp = 0;
-            startupDetectorTemperatureDegC = (short)temp;
-            if (startupDetectorTemperatureDegC >= 99)
-                startupDetectorTemperatureDegC = 15;
-            else if (startupDetectorTemperatureDegC <= -50)
-                startupDetectorTemperatureDegC = 15;
+            TECSetpoint = (short)temp;
+            if (TECSetpoint >= 99)
+                TECSetpoint = 15;
+            else if (TECSetpoint <= -50)

```

Comment:
Note that Andor can go below -50 but okay.

Reply:
I believe this is a "give me sane defaults" function, which the Andor EEPROM then immediately overwrites, but I take your point. This only should "stick" if the EEPROM fails to read and that shouldn't be possible for Andor since there is no physical read